common-react/@dbeaver/ui-kit/src/Button/button.css

```diff
@@ -0,0 +1,164 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2025 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+@layer components {
```

Comment:
file name is incorrect (must be **B**utton)

Reply:
git didn't want to change it) Fixed.